### PR TITLE
adding extra-headers to embedding

### PR DESCRIPTION
**Pull request by ali-arian**
## Adding extra-headers to Embedding

This PR fixes the issue with Embedding not passing the `extra-headers` for azure calls.

## Type

🆕 New Feature
🐛 Bug Fix
🧹 Refactoring
📖 Documentation
🚄 Infrastructure
✅ Test

## [REQUIRED] Testing - Attach a screenshot of any new tests passing locall
If UI changes, send a screenshot/GIF of working UI fixes

### CODE REVIEW

**Comment by ali-arian** [comment]
@krrishdholakia can you please take a look when you got a chance?

**Comment by ali-arian** [comment]
@ishaan-jaff can you please take a look when you got a chance?

**Comment by ali-arian-bread** [comment]
@ishaan-jaff can you please take a look when you got a chance? This is blocking us to use the official image.

**Comment by ali-arian** [comment]
@ishaan-jaff can you please take a look when you got a chance? This is blocking us to use the official image.

File: litellm/main.py [hunk 1]
```diff
@@ -3387,6 +3387,7 @@ def embedding(  # noqa: PLR0915
                 optional_params=optional_params,
                 client=client,
                 aembedding=aembedding,
+                extra_headers=extra_headers,
```
Comment (ghost) [inline litellm/main.py:3390]: this is not necessary. extra_headers is already passed via `optional_params`

**Comment by ghost** [comment]
Hey @ali-arian extra_headers is already supported on embedding - https://github.com/BerriAI/litellm/blob/c73ce95c01d33eeebe8adb4df2b899053261e405/litellm/main.py#L3390


which is passed in the request here - 
https://github.com/BerriAI/litellm/blob/c73ce95c01d33eeebe8adb4df2b899053261e405/litellm/llms/AzureOpenAI/azure.py#L881

**Pull request closed without merging**